### PR TITLE
MERGE-AFTER-SIMP-6.3.2 (SIMP-952) Update tftpboot regarding disk crypto

### PR DESCRIPTION
**Pull request by trevor-vaughan**
- Updated tftpboot documentation to reference the Disk Encryption capability in
  the kickstart files.
- Fixed moved links

SIMP-952 #comment Update tftpboot documentation

### CODE REVIEW

File: docs/user_guide/PXE_Boot.inc [hunk 1]
```diff
@@ -201,9 +201,17 @@ various files to model different systems.
      kickstart file needed.
 
      .. NOTE::
-        If using the default cfg files, know that they do not have the '_el[6,7]' 
+        If using the default cfg files, know that they do not have the '_el[6,7]'
         tags at the end of their name.
```
Comment (sharkbruhaha) [inline docs/user_guide/PXE_Boot.inc]: Should be "names" to follow plurality of the rest of the sentence?

Comment (trevor-vaughan) [reply docs/user_guide/PXE_Boot.inc]: @sharkbruhaha If you feel that minor syntactical changes are necessary, can you suggest them via the GitHub editor?

File: docs/user_guide/PXE_Boot.inc [hunk 2]
```diff
@@ -217,7 +225,7 @@ various files to model different systems.
             kernel => 'OSTYPE-MAJORRELEASE-ARCH/vmlinuz',
             initrd => 'OSTYPE-MAJORRELEASE-ARCH/initrd.img',
             ks     => "https://KSSERVER/ks/pupclient_x86_64_el7.cfg",
-            extra  => "inst.noverifyssl ksdevice=bootif\nipappend 2"
+            extra  => "inst.noverifyssl ksdevice=bootif simp_disk_crypt\nipappend 2"
```
Comment (lnemsick-simp) [inline docs/user_guide/PXE_Boot.inc]: This did not work as expected.   I was prompted for the passphrase (which I didn't know) after the client kickstarted.  For a KS client, is other setup required?

Comment (trevor-vaughan) [reply docs/user_guide/PXE_Boot.inc]: @lnemsick-simp Does this work now with the fixes in https://github.com/simp/simp-core/pull/592? Looks like this was actually a long-standing bug.

Comment (lnemsick-simp) [reply docs/user_guide/PXE_Boot.inc]: Yes, the bug fix addresses the problem.

**Pull request merged**